ibis/backends/duckdb/datatypes.py

```diff
@@ -63,9 +64,14 @@ def parse(text: str, default_decimal_parameters=(18, 3)) -> DataType:
         | spaceless_string("double", "float8").result(float64)
         | spaceless_string("real", "float4", "float").result(float32)
         | spaceless_string("smallint", "int2", "short").result(int16)
-        | spaceless_string("timestamp", "datetime").result(
-            Timestamp(timezone="UTC")
-        )
+        | spaceless_string(
+            "timestamp_tz",
+            "timestamp_sec",
+            "timestamp_ms",
```

Comment:
Not preserving the scale in the conversion at all?

Reply:
We don't have a way to do that yet in ibis. Currently, it doesn't have any meaningful affect on the APIs we present to users, so it shouldn't be a huge deal. We are considering adding an optional resolution/scale property to our `Timestamp` type for better compatibility with backends that support it but that'll be a separate PR.

Reply:
The plan is to have something similar to the way our `Decimal` type works: backends that care about scale can use/create such information in the type, otherwise the scale value is `None` meaning no scale or unknown scale.